ngraph/core/include/openvino/core/function.hpp

```diff
@@ -24,6 +25,12 @@
 #include "openvino/runtime/tensor.hpp"
 
 namespace ov {
+#ifdef NGRAPH_UNIT_TEST_ENABLE
```

Comment:
Why do we need to have this test dependency?
I don't want to have such dependencies in our public headers.

Reply:
Decided to keep ov::FuncionAccessor without macro until further better solutions.